test/runtests.jl

```diff
@@ -4,27 +4,29 @@ using UnROOT
 nthreads = UnROOT._maxthreadid()
 nthreads == 1 && @warn "Running on a single thread. Please re-run the test suite with at least two threads (`julia --threads 2 ...`)"
 
-include("Aqua.jl")
```

Comment:
this change has made tests printing delayed until something goes wrong, feels a bit annoying